OPAL/tac/src/main/scala/org/opalj/tac/cg/android/AndroidICCAnalysis.scala

```diff
@@ -669,7 +667,10 @@ class AndroidICCAnalysis(val project: SomeProject) extends FPCFAnalysis {
                 //Intent(String action)
                 if (p.head.asVar.value.verificationTypeInfo == ObjectVariableInfo(ObjectType.String)) {
                     val intent = new ImplicitIntent(m)
-                    intent.action = p.head.asVar.value.toString.replaceAll(paramRegex, "")
+                    val defSite = p.head.asVar.definedBy.head
```

Comment:
Are you absolutely sure that checking only the first definition site (`head`) is sufficient here? Usually, one has to check all definition sites of a variable (including some handling of the case that it is unknown because it is not inside the method, i.e., it is <0).